persistence/postgres-persistence/src/main/resources/db/migration_postgres/V8__outbox_schema.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE outbox_table (
+  id SERIAL,
+  entity_id varchar(255) NOT NULL,
+  info TEXT,
+  status varchar(255) NOT NULL,
+  PRIMARY KEY (id, status)
```

Comment:
shouldn't this be entity_id + status ? 

do we need both the ID and entity_id ?